core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -133,12 +133,12 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with PrivateMethodTe
     val bmSecurityMgr = new SecurityManager(bmConf, encryptionKey)
     val serializerManager = new SerializerManager(serializer, bmConf, encryptionKey)
     val transfer = transferService.getOrElse(new NettyBlockTransferService(
-      conf, securityMgr, serializerManager, "localhost", "localhost", 0, 1))
+      bmConf, securityMgr, serializerManager, "localhost", "localhost", 0, 1))
```

Comment:
Is this an improvement to add a new test case or a bug, @Ngone51 ?

Reply:
It's a bug. `makeBlockManager` should use `testConf` instead of global `conf` when `testConf` exists.